full/Dockerfile

```diff
@@ -249,29 +244,14 @@ RUN echo "rvm_gems_path=/workspace/.rvm" > ~/.rvmrc
 LABEL dazzle/layer=lang-rust
 LABEL dazzle/test=tests/lang-rust.yaml
 USER gitpod
-RUN sudo apt-get update \
-    && DEBIAN_FRONTEND=noninteractive sudo apt-get install -yq \
-        # Enable Rust static binary builds
-        musl \
-        musl-dev \
-        musl-tools \
-    && sudo cp /var/lib/dpkg/status /var/lib/apt/dazzle-marks/lang-rust.status \
-    && sudo apt-get clean \
-    && sudo rm -rf /var/lib/apt/lists/* /tmp/*
-
 RUN cp /home/gitpod/.profile /home/gitpod/.profile_orig && \
-    curl -fsSL https://sh.rustup.rs | sh -s -- -y \
-    && .cargo/bin/rustup toolchain install 1.48.0 \
-    && .cargo/bin/rustup default 1.48.0 \
-    # Save image size by removing now-redudant stable toolchain
-    && .cargo/bin/rustup toolchain uninstall stable \
+    curl -fsSL https://sh.rustup.rs | sh -s -- -y --profile minimal --default-toolchain 1.48.0 \
```

Comment:
The `minimal` installation without `musl` should be tested with an actual Rust project e.g. Nushell.

Reply:
Tested with Nushell, works well!